src/test/java/attendance/model/domain/crew/AbsenceCountTest.java

```diff
@@ -0,0 +1,21 @@
+package attendance.model.domain.crew;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class AbsenceCountTest {
+  @Test
+  void 결석_횟수는_음수가_될_수_없습니다() {
+    Assertions.assertThatThrownBy(() -> {
+          AbsenceCount.from(-1);
```

Comment:
```suggestion
    Assertions.assertThatThrownBy(() -> AbsenceCount.from(-1))
```
`{`를 사용할 필요가 없습니다. intellij 에서는 대부분 안티패턴이나 무의미한 코드에 대해서 특정한 메시지를 남기니 의식하면서 개발해보시면 좋을 것 같아요.

Reply:
세심한 리뷰 감사드립니다 😊

미쳐 신경쓰지 못하고 넘어갔던 부분들인데, 더 꼼꼼히 메시지를 인식하면서 개발해 보아야겠네요!